smartsim/_core/utils/telemetry/manifest.py

```diff
@@ -229,7 +245,7 @@ def load_manifest(file_path: str) -> t.Optional["RuntimeManifest"]:
             raise ValueError("Manifest missing required runs")
 
         exp_dir = pathlib.Path(exp["path"])
-        runs = [Run.load_run(raw_run, exp_dir) for raw_run in runs]
+        runs = [Run.load_run(raw_run, exp_dir, manifest_dict) for raw_run in runs]
```

Comment:
I'm not sure how I feel about shallow copying `runs` and `exp_dir` out of `manifest_dict`  but then passing `manifest_dict` around the everywhere alongside the copied data. Can we just pass around the `manifest_dict` instead??

Reply:
I wanted to give the deserialization the object it was working on but also let it see higher level context info when it needed to do something wacky, like change behavior of  if it's a dragon launcher. I didn't like sending `is_dragon` all the way down, nor do I like the idea of having to add a `launcher: str` param, then another later, etc...

I think I like switching to passing only the experiment. That achieves my goal but avoids passing the entire manifest around. What do you think of that?

Reply:
That makes sense to me, and resolves my "multiple parameters pointing to same data" complaint! Totally in favor!!